Sources/PaymentButtons/PaymentButtonFont.swift

```diff
@@ -1,12 +1,15 @@
 import UIKit
 
 enum PaymentButtonFont {
-    /// The primary font to be employed in the UI for non special circumstances
-    public static let primaryFont: UIFont = .systemFont(ofSize: UIFont.systemFontSize)
+    /// The primary font
+    public static let paypalPrimaryFont = UIFont(name: "PayPalOpen-Regular", size: 14) ??
```

Comment:
It looks like we aren't using the `-Medium` or `-Bold` font options? Are we planning to at some point?

Reply:
Not that I'm aware of unless they'll be used in other parts of the SDK. I'll remove for now and we can always add them in the future as needed

Reply:
Actually, since we're changing to 2 button sizes and not incorporating taglines, we only need primary font. Otherwise the other font sizes were needed for different button sizes and tagline